ansible.cfg

```diff
@@ -2,6 +2,16 @@
 host_key_checking = False
 retry_files_enabled = False
 roles_path = ~/.ansible/roles/opensrp:./roles
+#
+# Use Mitogen connection backend by default for faster execution
+# See: https://mitogen.networkgenomics.com/ansible_detailed.html
+#strategy_plugins = .venv/lib/python3.7/site-packages/ansible_mitogen/plugins/strategy
+strategy = mitogen_linear
```

Comment:
@ukanga we might need to add the Mitogen dependency in the https://github.com/OpenSRP/playbooks/blob/master/requirements/base.{in,pip} files.

Reply:
Thank you @jasonrogena